README.rst

```diff
@@ -314,6 +314,25 @@ This may be useful for Python programs in which frequent flushing of write updat
 with descriptors on such objects possibly being held open for indeterminately long lifetimes -- yet the eventual
 application of those updates prior to the teardown of the Python interpreter is required.
 
+The current value of the setting is global in scope (ie applies to all sessions, whenever created) and is always
+consulted for the creation of any data object handle to govern that handle's cleanup behavior.
+
+Python iRODS Client Settings File
+---------------------------------
+
+As of v1.1.9, Python iRODS client configuration can be saved in, and loaded from, a settings file.
+
+The use of a settings file can be indicated, and the path to that file determined, by setting the environment variable:
+:code:`PYTHON_IRODSCLIENT_CONFIGURATION_PATH`.  If this variable is present but empty, this denotes use of a default settings
+file path of :code:~/.python-irodsclient`;  if the variable's value is of nonzero length, the value should be an absolute path
+to the settings file whose use is desired.  Also, if the variable is set, auto-load of settings will be performed, meaning
+that the act of importing :code:`irods` or any of its submodules will cause the automatic loading the settings from the
+settings file, assuming it exists.  (Failure to find the file at the indicated path will be logged as a warning.)
```

Comment:
should it be an error?  to make sure the users knows they are not running what they specified?

Reply:
I prefer it as a warning, because users may set the env variable for autoload knowing they are going to be save( )'ing to that path.  It isn't fatal, and if you're an interested enough party you'll be reading the warnings.... right?

Reply:
sounds fair.    leave it.